<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>GlobalSettingsPage</class>
 <widget class="QWidget" name="GlobalSettingsPage">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>400</width>
    <height>339</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <widget class="QGroupBox" name="groupBox">
     <property name="title">
      <string>Системы координат</string>
     </property>
     <layout class="QHBoxLayout" name="horizontalLayout">
      <item>
       <layout class="QFormLayout" name="formLayout">
        <item row="0" column="0">
         <widget class="QLabel" name="Label">
          <property name="text">
           <string>Система координат</string>
          </property>
         </widget>
        </item>
        <item row="0" column="1">
         <widget class="QComboBox" name="coordSystem"/>
        </item>
        <item row="1" column="1">
         <widget class="QComboBox" name="coordFormat"/>
        </item>
        <item row="1" column="0">
         <widget class="QLabel" name="Label_2">
          <property name="text">
           <string>Формат координат</string>
          </property>
         </widget>
        </item>
       </layout>
      </item>
     </layout>
    </widget>
   </item>
<!--
   <item>
    <widget class="QGroupBox" name="lcsSettings">
     <property name="title">
      <string>Настройки локальной системы координат</string>
     </property>
     <layout class="QVBoxLayout" name="lcsSettingsLayout">
      <item>
       <layout class="QHBoxLayout" name="lcsMethodLayout">
        <item>
         <widget class="QLabel" name="lcsMethodLabel">
          <property name="text">
           <string>Способ задания (WGS84)</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QComboBox" name="lcsMethod">
          <property name="sizePolicy">
           <sizepolicy hsizetype="MinimumExpanding" vsizetype="Fixed">
            <horstretch>0</horstretch>
            <verstretch>0</verstretch>
           </sizepolicy>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item>
       <layout class="QGridLayout" name="lcsLayout">
        <item row="1" column="3" rowspan="2">
         <widget class="QToolButton" name="lcsCenterCopy">
          <property name="text">
           <string/>
          </property>
         </widget>
        </item>
        <item row="1" column="1">
         <widget class="QLabel" name="lcsCenterLatitudeLabel">
          <property name="text">
           <string>Широта:</string>
          </property>
          <property name="alignment">
           <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
          </property>
         </widget>
        </item>
        <item row="0" column="7">
         <spacer name="horizontalSpacer1">
          <property name="orientation">
           <enum>Qt::Horizontal</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>10</width>
            <height>5</height>
           </size>
          </property>
         </spacer>
        </item>
        <item row="2" column="8">
         <widget class="QLabel" name="lcsDirectionLongitudeLabel">
          <property name="text">
           <string>Долгота:</string>
          </property>
          <property name="alignment">
           <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
          </property>
         </widget>
        </item>
        <item row="0" column="5">
         <spacer name="horizontalSpacer0">
          <property name="orientation">
           <enum>Qt::Horizontal</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>10</width>
            <height>5</height>
           </size>
          </property>
         </spacer>
        </item>
        <item row="3" column="8">
         <widget class="QLabel" name="lcsDirectionAngleLabel">
          <property name="text">
           <string>Угол:</string>
          </property>
          <property name="alignment">
           <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
          </property>
         </widget>
        </item>
        <item row="0" column="1" colspan="4">
         <widget class="QLabel" name="lcsCenterLabel">
          <property name="text">
           <string>Начало координат</string>
          </property>
          <property name="alignment">
           <set>Qt::AlignCenter</set>
          </property>
         </widget>
        </item>
        <item row="3" column="1">
         <widget class="QLabel" name="lcsCenterAltitudeLabel">
          <property name="text">
           <string>Высота:</string>
          </property>
          <property name="alignment">
           <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
          </property>
         </widget>
        </item>
        <item row="0" column="6" rowspan="4">
         <widget class="Line" name="line">
          <property name="orientation">
           <enum>Qt::Vertical</enum>
          </property>
         </widget>
        </item>
        <item row="0" column="12">
         <spacer name="horizontalSpacer3">
          <property name="orientation">
           <enum>Qt::Horizontal</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>10</width>
            <height>5</height>
           </size>
          </property>
         </spacer>
        </item>
        <item row="1" column="4" rowspan="2">
         <widget class="QToolButton" name="lcsCenterPaste">
          <property name="text">
           <string/>
          </property>
         </widget>
        </item>
        <item row="2" column="1">
         <widget class="QLabel" name="lcsCenterLongitudeLabel">
          <property name="text">
           <string>Долгота:</string>
          </property>
          <property name="alignment">
           <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
          </property>
         </widget>
        </item>
        <item row="1" column="8">
         <widget class="QLabel" name="lcsDirectionLatitudeLabel">
          <property name="text">
           <string>Широта:</string>
          </property>
          <property name="alignment">
           <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
          </property>
         </widget>
        </item>
        <item row="0" column="0">
         <spacer name="horizontalSpacer2">
          <property name="orientation">
           <enum>Qt::Horizontal</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>10</width>
            <height>5</height>
           </size>
          </property>
         </spacer>
        </item>
        <item row="0" column="8" colspan="4">
         <widget class="QLabel" name="lcsDirectionLabel">
          <property name="text">
           <string>Направление</string>
          </property>
          <property name="alignment">
           <set>Qt::AlignCenter</set>
          </property>
         </widget>
        </item>
        <item row="1" column="10" rowspan="2">
         <widget class="QToolButton" name="lcsDirectionCopy">
          <property name="text">
           <string/>
          </property>
         </widget>
        </item>
        <item row="1" column="11" rowspan="2">
         <widget class="QToolButton" name="lcsDirectionPaste">
          <property name="text">
           <string/>
          </property>
         </widget>
        </item>
       </layout>
      </item>
     </layout>
    </widget>
   </item>
-->
   <item>
    <widget class="QGroupBox" name="groupBox_2">
     <property name="title">
      <string>Командно-пилотажный прибор</string>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout_2">
      <item>
       <widget class="QCheckBox" name="invertedPfdIndication">
        <property name="text">
         <string>Обратная индикация авиагоризонта</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QCheckBox" name="showOnMapPfd">
        <property name="text">
         <string>Отображать поверх карты</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <spacer name="verticalSpacer">
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>20</width>
       <height>40</height>
      </size>
     </property>
    </spacer>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
 <designerdata>
  <property name="gridDeltaX">
   <number>5</number>
  </property>
  <property name="gridDeltaY">
   <number>5</number>
  </property>
  <property name="gridSnapX">
   <bool>false</bool>
  </property>
  <property name="gridSnapY">
   <bool>false</bool>
  </property>
  <property name="gridVisible">
   <bool>false</bool>
  </property>
 </designerdata>
</ui>
